2-mandatory/2-financial-times.js

```diff
@@ -24,6 +34,17 @@ function titleWithFewestWords(allArticleTitles) {
 */
```

Comment:
Looks like this works!
If you get time, you can experiment with different ways of checking if there's a number in a string - and then have a think about whether one approach might be better than another.